source/org/zfin/nomenclature/repair/GenotypeNamingIssues.java

```diff
@@ -74,9 +76,31 @@ public void categorizeAlphabeticalTranspositions(List<NamingIssuesReportRow> row
         }
     }
 
+    public void categorizeManuallyApprovedChanges(List<NamingIssuesReportRow> rows) {
+        //see:https://docs.google.com/spreadsheets/d/1DW28aF66zDr6g9oqtFfV_Xwwseeb0DMP_uDz61BHsbI/edit
+        Map<String, String> fixes = new HashMap<>();
+        fixes.put("cct5<sup>tf212</sup>", "cct5<sup>tf212b</sup>");
```

Comment:
I changed it to accept a file upload via the jenkins task instead of having hard-coded fixes.  

https://user-images.githubusercontent.com/90803397/154170436-4e49ed9d-93ea-4231-856d-c7f56507cf42.mp4